docs/UserGuide.md

```diff
@@ -323,7 +323,7 @@ Format: `addClient n/NAME p/PHONE_NUMBER [a/ADDRESS] [e/EMAIL] [b/BIRTHDAY] [pd/
 * `PHONE_NUMBER` should contain only numbers and be at least 8 digits long.
 * `EMAIL`, `BIRTHDAY`, `ADDRESS` and `PRODUCT` are optional.
 * `BIRTHDAY` in the future are not acceptable.
-* `PRODUCT` must exists already.
+* `PRODUCT` must exists already. Reuse `pd\` prefix to add each product to the client.
```

Comment:
I think this description is not clear enough. A better way could be `You can add multiple products to the client`.
Then add one example to show how the user can add multiple products. Please do the same for `editClient` as well.

Reply:
Ok, made the changes. Please have a look, thank you.